process/smartContract/process.go

```diff
@@ -291,7 +291,7 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 		return nil
 	}
 
-	sc.txFeeHandler.ProcessTransactionFee(feeForValidators)
+	sc.txFeeHandler.ProcessTransactionFee(feeForValidators, nil)
```

Comment:
Every call of ProcessTransactionFee method is done with nil parameters in whole project Why?

Reply:
fixed.